janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManager.java

```diff
@@ -157,6 +160,7 @@ public byte getByte() {
 
     public CassandraThriftStoreManager(Configuration config) throws BackendException {
         super(config);
+        log.warn("Thrift is deprecated and will be removed with JanusGraph 0.5.0. Please switch to the CQL backend.");
```

Comment:
s/Thrift/Cassandra Thrift protocol/ ?

Basically, please include "Cassandra" here somewhere so that when this is out-of-context in the logs, it's obvious what "Thrift" refers to as it's a separate Apache top-level project (https://thrift.apache.org/) and it's not deprecated on its own.